src/packages/src/xroad/ubuntu/generic/xroad-center.postinst

```diff
@@ -229,10 +229,23 @@ configure | reconfigure)
     chmod 755 /var/lib/xroad/import
     chown xroad:xroad /var/lib/xroad/import
 
-    invoke-rc.d --quiet rsyslog try-restart || true
-    invoke-rc.d --quiet nginx try-restart || true
-    invoke-rc.d --quiet xroad-signer try-restart || true
-    invoke-rc.d --quiet xroad-jetty restart || true
+    RET=
+    db_get xroad-common/skip-cs-db-migrations || true
+    skip_migrations="${RET}"
+
+    if [ "$skip_migrations" = false ] ; then
+      invoke-rc.d --quiet rsyslog try-restart || true
+      invoke-rc.d --quiet nginx try-restart || true
+      invoke-rc.d --quiet xroad-signer try-restart || true
+      invoke-rc.d --quiet xroad-jetty restart || true
+      echo "services restarted"
+    else
+      invoke-rc.d --quiet rsyslog stop || true
+      invoke-rc.d --quiet nginx stop || true
+      invoke-rc.d --quiet xroad-signer stop || true
+      invoke-rc.d --quiet xroad-jetty stop || true
+      echo "services remain stopped since the database migrations were skipped"
```

Comment:
It would be sufficient to leave just the xroad-jetty stopped, other services do not depend on the database

Reply:
Fixed in https://github.com/nordic-institute/X-Road/pull/1024/commits/db63a2172197f094aa53dcd65a2e523d2df19fb2